prow-configurations/README.md

```diff
@@ -51,12 +51,15 @@ Copy the address of the ingress `ing` and open it in a browser to display the Pr
 
 ## Configuration
 To allow sending events from Github repository to Prow, configure Webhook as described [here](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#add-the-webhook-to-github). 
-Prow can be configured by specifying plugins. To generate the `plugins.yaml` file, use `./development/generate.sh`. 
-`Generate.sh` combines template file (`plugins.yaml.tpl`) with actual values provided as a JSON file (environment variable `INPUT_JSON`) 
-and generates output to `plugins.yaml` file. 
-To check if configuration files `plugins.yaml` and `config.yaml` are correct, execute the `development/check.sh` script.
-In case of no errors, use the `update-plugins.sh` script to update plugins configuration on cluster. 
+You can configure Prow by specifying two files: `plugins.yaml` and `config.yaml`. To generate them, use `./development/generate.sh`. 
```

Comment:
You can configure Prow by specifying ``the `plugins.yaml` and `config.yaml` files.``